### PR TITLE
RevIssuer works in one case

### PR DESCRIPTION
**Pull request by dckc**
It worked nearly first try!

This is a manual translation from [RevIssuer.js][1] to rholang.

When I plug it into [Liquid Democracy Participation Experiment](https://rgov1.netlify.app/) and do a signed deploy (on testnet) I get [results like this][2].

[1]: https://github.com/rchain-community/js2rho/blob/jessica/examples/RevIssuer.js
[2]: https://gist.github.com/dckc/2c8d3daec88a782b0892c4c77e0c542c

### CODE REVIEW

File: RevIssuer.rho [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            }
+          }
+        }
+      }
+    } |
+
+    contract RevIssuerKit(ret) = {
+      new mint, issuer /* TODO: amountMath, brand */ in {
+
+        ret!({"mint": *mint, "issuer": *issuer}) |
```
Comment (dckc) [inline RevIssuer.rho:47]: oops... there is no mint.

**Comment by dckc** [comment]
It's deployed to testnet:

```
      [
        "tests passed; inserted RevIssuerKit at",
        "rho:id:mybn43dykzf3sgmgy165ipcaexa1h37mdp1rrksdb7ucjpteprnnmp",
        "REV issuer at",
        "rho:id:1xm16jeazfxpyyd7a4834st854nsjgg8izdzbuagc7cigcd3xu3nqz"
      ]
```

**Comment by dckc** [comment]
refinement 1c442a8 with `getDepositFacet` deployed...

```
      [
        "tests passed; inserted RevIssuerKit at",
        "rho:id:frbg3nokk9qqk37jw367zataqkhd4bz7jmoou5wztimybh48o1cfym",
        "REV issuer at",
        "rho:id:uymahacuqbsqgwqcxxdo1eu8ake47uzsuomcq3gode6wuh179une77"
      ]
```

**Comment by dckc** [comment]
critical fix! 78f3716

```
      [
        "tests passed; inserted RevIssuerKit at",
        "rho:id:k8o43dz9troqqs3mhu8e8ah49mhs4a1i8r8bn17en6mhek5mffpyci",
        "REV issuer at",
        "rho:id:j38i5p3f64fcda75gh1sx43friatxzn9td6sdh1jajomg97k7wbhj3"
      ]
```

**Review by jimscarver — APPROVED** [review]

**Comment by dckc** [comment]
another fix:

```
      [
        "tests passed; inserted RevIssuerKit at",
        "rho:id:akhfr6c37ywjt7zfn8kzcxy6qx1no6jxezb85nm143nap8ienchd4h",
        "REV issuer at",
        "rho:id:961h8n36ikxz4yxxfye898wh7or3d37fbp1yddmsei1rw64hcyzjmf"
      ],
```

**Pull request merged**